src/olympia/blocklist/cron.py

```diff
@@ -21,12 +22,15 @@ def upload_mlbf_to_kinto():
         KINTO_BUCKET, KINTO_COLLECTION_MLBF, kinto_sign_off_needed=False)
     stats = {}
     key_format = get_mlbf_key_format()
+    generation_time = int(time.time() * 1000)
```

Comment:
Is this temporary (until you have a better way of finding the time), or permanent?

In any case, a comment about the significance of this time stamp would be nice. Even if it is just a reference to #13695.

Reply:
The conclusion from #13695 was the time on AMO was for all practical purposes the same time as on autograph so this is the best way, unless I'm misunderstanding the other's comments.  If you still disagree I'd comment in the issue.  

I'll add a comment in the code.

Reply:
https://github.com/mozilla/addons-server/pull/13749/commits/c8379bbe5dd733744597b37e1446b80f7dd1b6cd